pkg/minikube/constants/constants.go

```diff
@@ -35,6 +35,8 @@ const (
 	DefaultCPUS                  = 2
 	DefaultDiskSize              = "20g"
 	UpdateMarkerFileName         = "updated"
+	OPENSHIFT_CONTAINER_NAME     = "origin"
```

Comment:
@hferentschik We do need to move it somewhere otherwise we can't import addon package to openshift package level because of cyclic dependencies not allowed. I do agree let's not use Minikube directory but we can have a constants for openshift pkg level.